### ISSUE TITLE
Results not as good as expected

### ISSUE THREAD

**Issue by cardosofelipe** [cotton6/COTTON-size-does-matter#11]
First things first: thank you for the great work, your project is one of the best open-sourced virtual-try-on via AI implementations I have seen so far. Despite the results I'm getting, I am still very impressed by the amount of work you put on it and the amount of material released.
Now, back on the issue.
In the last few days I am trying to test your model against some custom data. Initially I struggled to avoid some "bad" pictures for the pre-processing scripts, however I managed to obtain some model/product pairs and to build a small dataset to test the capabilities of such model. Some results are a bit weird, for instance,
this product:
![5](https://github.com/cotton6/COTTON-size-does-matter/assets/17471456/f28f8014-2f3c-429d-92e7-1d1f5ecab257)

on this model:
![7](https://github.com/cotton6/COTTON-size-does-matter/assets/17471456/97b3c518-e5e8-49fb-b9b1-587bcdf73024)

gives me this result:
![5_5](https://github.com/cotton6/COTTON-size-does-matter/assets/17471456/ae8584ab-26f8-42e2-a1b9-17f757a87113)

The example above is one of the best results I got so far and it has several problems, but the one I would like to address with this issue is the texture. It seems that almost always I get a knitwear-like texture, even though the input piece of cloth is not one. These are some additional results:
![7_5](https://github.com/cotton6/COTTON-size-does-matter/assets/17471456/165239f9-16a6-44d1-9065-4d85969534c9)
![6_0](https://github.com/cotton6/COTTON-size-does-matter/assets/17471456/7ab3b4de-2213-4250-94dd-600a137f2e69)

Basically, not matter what kind of cloth I try, I always get a wrong texture, very similar to the ones shown above.

**Technical Details:**

- Image Format: JPG
- Image Dimensions: Width = 768, Height = 1024

**Questions:**

- Could the issue stem from the images I am using?
- Is there a potential bias in the model's weights?
- My inclination is to retrain the model with new data. Has this approach been attempted in response to similar issues?

I would greatly appreciate any suggestions or guidance on how to address this texture representation issue. Thank you for your time and assistance.

**Comment by ChiehYunChen** [comment]
Hi @cardosofelipe, 

Thank you for sharing your experience.

The issue might be the checkerboard effect because our architecture contains "transposed convolution.” 
If you have other cool findings, we appreciate knowing them!

**Comment by StartHua** [comment]
Me too, have you solved it?
